src/gtk/animate.cpp

```diff
@@ -284,8 +253,19 @@ void wxAnimationCtrl::SetAnimation(const wxAnimation &anim)
     ResetAnim();
     ResetIter();
 
+    m_animation = anim;
```

Comment:
Why do we use `m_animation` here? This is the base (generic) class member, so I think we shouldn't touch it at all when we're not using the generic implementation.

Reply:
It still feels a bit dirty to reuse the base class object like this here, but I guess it's ok then... To preserve what the current code does we'd need to return `wxAnimation(new wxAnimationImplGTK(m_anim))`, I guess?

Reply:
Yes, or something like that.